lib/VVRestApi/VVRestApiNodeJs/config.yml

```diff
@@ -40,7 +40,7 @@ ResourceUri:
   UserSupervisors: /users/{id}/supervisors
   UserSupervisees: /users/{id}/supervisees
   UserById: /users/{id}
-  UserDelete: /customerdatabase/{databaseId}/deleteuser/{AuthenticationUserId}
+  UserDelete: /customerdatabase/{databaseId}/deleteuser/{authenticationUserId}
```

Comment:
Can we rename this `CustomerDatabaseRemoveUser `and move it under `CustomerDatabaseAssignUser` to keep with the convention?